docs/developer_guide.md

```diff
@@ -19,14 +19,20 @@ the repository, and create a pull request.
 
 1.  Create a fork of the [XLA repository](https://github.com/openxla/xla).
 2.  Clone your fork of the repo, replacing `<USER>` with your GitHub username:
-    ```sh
-    git clone https://github.com/<USER>/xla.git
-    ```
+    <pre class="devsite-click-to-copy">
```

Comment:
This was rendering an HTML link syntax in the code snippet, hence this update.

I found this syntax in:
- https://developers.google.com/style/code-samples#basic-guidelines
- https://github.com/tensorflow/docs/blob/15e96c3ee154d7c10ec9bb807ef44b4e6f08e65b/site/en/install/source.md?plain=1